build.sbt

```diff
@@ -462,6 +463,15 @@ lazy val cborJson = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     tlVersionIntroduced := Map("3" -> "1.5.1", "2.13" -> "1.5.1", "2.12" -> "1.5.1")
   )
 
+lazy val msgpack = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Full)
```

Comment:
If all the sources live in the `shared` directory, let's remove this one level of indirection by using `CrossType.Pure` and move the `src` directory at the top.